common/app/views/fragments/items/facia_cards/item.scala.html

```diff
@@ -17,11 +18,18 @@
 
 @defining(containerIndex == 0) { case (isFirstContainer) =>
     @item match {
+
         case content: ContentCard => {
             @contentCard(content, containerIndex, index, card.visibilityDataAttribute, isFirstContainer, isList)
         }
 
-        case paidCard: PaidCard => {
+        case paidContentOnPaidPage: PaidCard if frontProperties.exists(_.isPaidContent) && paidContentOnPaidPage.contentCard.isDefined =>{
+            @paidContentOnPaidPage.contentCard.map { c =>
+                @contentCard(c, containerIndex, index, card.visibilityDataAttribute, isFirstContainer, isList)
```

Comment:
there are other differences between the templates besides the *sponsored by* text. I would rather not copy/paste the template as then they may diverge without people considering the paid cards. Right now, they are simply a normal content card reskinned by CSS. CC @lps88